templates/repo/commits_list_small.tmpl

```diff
@@ -13,6 +13,8 @@
 			{{avatarByEmail $.root.Context .Author.Email .Author.Name}}
 		{{end}}
 
+		{{$commitLink:= printf "%s/commit/%s" $.comment.Issue.PullRequest.BaseRepo.Link (PathEscape .ID.String)}}
+
```

Comment:
I think it's not that far. The code block looks like this:

```
{{$commitLink:=....}}
<span class="ui float right shabox">{{$commitLink}}</span>  ## The Line 18
<span class="gt-mono commit-summar"..>{{$commitLink}}</span> ## The Line 50
```

So you can see that now `commitLink` is at the same level of other blocks.

If we put it below line 34, it would look like

```
<span class="ui float right shabox">
    {{$commitLink := ...}}
    {{$commitLink}}
</span>
<span class="gt-mono commit-summar"..>
    {{$commitLink}}
</span>
```

It looks like crossed the code blocks, it doesn't look good to me.